packages/components/src/components/badges/badge.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, Prop, h } from '@stencil/core';
+import { CssClassMap } from '../../utils/utils';
+import classNames from 'classnames';
+
+@Component({
+    tag: 't-badge',
+    styleUrls: [
+        'badge.css'
+    ],
+    shadow: true
+})
+
+export class Badge {
+    @Prop() size?: string = '';
+
+
+    private getCssClassMap(): CssClassMap {
+        return classNames(
+            'badge',
+            this.size && `badge--size-${this.size}`
```

Comment:
'pill' class handler is missing (boolean)

Reply:
solved